wheatley/row_generation/plain_hunt_generator.py

```diff
@@ -9,6 +9,10 @@
 class PlainHuntGenerator(RowGenerator):
     """ A row generator to create plain hunt on any stage. """
 
+    def summary_string(self) -> str:
+        """ Returns a short string summarising the RowGenerator. """
+        return "Plain Hunt"
```

Comment:
I think this would work?
```suggestion
        return f"Plain Hunt on {self.stage}"
```

Reply:
Ah! Yes - I forgot that variable inheritance was a thing!

Reply:
Have rebased this change into a single commit.